### DISCUSSION TITLE
can someone find a shim for rammus for me?

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1263] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
can someone find a shim for rammus for me? shyvana rammus?

**Comment by schoolexploitkid** [comment, marked as answer]
found it on asus official website, download it here: https://www.asus.com/laptops/for-home/chromebook/asus-chromebook-c425/helpdesk_download?model2Name=ASUS-Chromebook-C425 (might make a shim library later)

**Comment by HelloIloveGimkit2** [comment]
what do you need it for
It might be somewhere or I can build it

**Reply by Balistic123** [reply]
anything that involves shims such as shimboot and br1ck. i have a chromebook flip c434ta shyvana rammus

**Reply by Balistic123** [reply]
> what do you need it for It might be somewhere or I can build it

i want to use br1ck and/or shimboot.  i have a chromebook flip c434ta shyvana rammus

**Reply by HelloIloveGimkit2** [reply]
aight go to one of my repos and create an issue

**Reply by HelloIloveGimkit2** [reply]
> > what do you need it for It might be somewhere or I can build it
> 
> i want to use br1ck and/or shimboot. i have a chromebook flip c434ta shyvana rammus

@Balistic123 what's your discord

**Reply by Balistic123** [reply]
> > > what do you need it for It might be somewhere or I can build it
> > 
> > 
> > i want to use br1ck and/or shimboot. i have a chromebook flip c434ta shyvana rammus
> 
> @Balistic123 what's your discord

I Can’t access discord because my school doesn’t allow it. Is there anything else you can use?

**Reply by HelloIloveGimkit2** [reply]
nope

**Reply by Balistic123** [reply]
> nope

ok ill give you my discord tonight

**Reply by HelloIloveGimkit2** [reply]
k

**Reply by Balistic123** [reply]
> k

poppopa1
